test/e2e/features/ginkgo/existing_tests.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Use existing ginkgo framework
+  As a test contributor
+  I want to not throw away all our old tests
+  In order to retain the value generated in them
+  @ginko @Conformance @NodeConformance @k8s.io
+  Scenario: Container Lifecycle Hook when create a pod with lifecycle hook should execute poststart exec hook properly
+    Given existing test "[k8s.io] Container Lifecycle Hook when create a pod with lifecycle hook should execute poststart exec hook properly [NodeConformance] [Conformance]"
```

Comment:
This is usually run with ```e2e.test --ginkgo.focus "\[k8s.io\] Container Lifecycle Hook when create a pod with lifecycle hook should execute poststart exec hook properly \[NodeConformance\] \[Conformance\]"```
I'm hoping we can find a way to run the test directly, sans before and aftersuite calls.